content/open-source/apko/faq.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "FAQs"
+type: "article"
+lead: "Frequently asked questions about apko"
+description: "Frequently asked questions about apko"
```

Comment:
I would delete description if it will be a repetition of lead as it may impact SEO